src/scss/components/_burger-menu.scss

```diff
@@ -0,0 +1,69 @@
+@keyframes slider-menu {
+  from {
+    left: -28rem;
+  }
+  to {
+    left: 0;
+  }
+}
+.hidden-menu {
+  display: none;
+}
+
+.wrapper__burger-menu {
+  background-color: #333;
+  opacity: 0.7;
+  position: fixed;
+  z-index: 999;
+  height: 100%;
+  min-height: 100vh;
+  transition: all 0.5s linear;
+}
+
+.burger-menu {
+  position: fixed;
+  background-color: $color-burger-menu-beckground;
+  width: 100%;
+  max-width: 70%;
+  height: 100%;
+  min-height: 100vh;
+  z-index: 1000;
```

Comment:
z-index: 1000 - please check all z-index in your styles. And refactor it to a realistic step (1, 2, 3 ...).